webui/react/src/pages/F_ExpList/glide-table/menu.tsx

```diff
@@ -37,19 +39,21 @@ export const TableActionMenu: React.FC<TableActionMenuProps> = ({
   handleClose,
   items,
 }) => {
+  const menuWidth = 200;
   const containerRef = useRef(null);
   useOutsideClickHandler(containerRef, handleClose);
+  const { width } = useResize();
 
   return (
     <div
       ref={containerRef}
       style={{
         border: 'solid 1px gold',
         display: !open ? 'none' : undefined,
-        left: x,
```

Comment:
@alexjdetermined 
This change is to fix that menu on last col gets cut off and isn't viewable